doc/gettingstarted.md

```diff
@@ -221,6 +221,22 @@ $ ttn-lw-cli end-devices create app1 dev2 \
 
 >Hint: You can also pass `--with-session` to have a session generated.
 
+It is also possible to create a multicast device (an ABP device which can not send uplinks and shares the security session with other devices) using the `--multicast` flag as follows:
+
+```bash
+$ ttn-lw-cli end-devices create app1 dev3 \
+  --frequency-plan-id EU_863_870 \
+  --lorawan-version 1.0.2 \
+  --lorawan-phy-version 1.0.2-b \
+  --abp \
+  --session.dev-addr 00E4304D \
+  --session.keys.app-s-key.key A0CAD5A30036DBE03096EB67CA975BAA \
+  --session.keys.nwk-s-key.key B7F3E161BC9D4388E6C788A0C547F255 \
+  --multicast
```

Comment:
@htdvisser shouldn't this be `multicast=true` ?

Reply:
Boolean flags can be set using just `--flag`, and they go to true by default.

```
adriansmares@A98BCD2222F1:~/go/src/go.thethings.network/lorawan-stack$ ttn-lw-cli end-devices create app1 dev2 \
>   --frequency-plan-id EU_863_870 \
>   --lorawan-version 1.0.2 \
>   --lorawan-phy-version 1.0.2-b \
>   --abp \
>   --session.dev-addr 00E4304D \
>   --session.keys.app-s-key.key A0CAD5A30036DBE03096EB67CA975BAA \
>   --session.keys.nwk-s-key.key B7F3E161BC9D4388E6C788A0C547F255 \
>   --multicast
Flag --session.keys.nwk-s-key.key has been deprecated, use the session.keys.f_nwk_s_int_key.key flag
 DEBUG Using access token (valid until 11:10AM)
 DEBUG ccResolverWrapper: sending update to cc: {[{localhost:1884 0  <nil>}] }
 DEBUG pickfirstBalancer: HandleSubConnStateChange: 0xc0005ddc00, CONNECTING
 DEBUG pickfirstBalancer: HandleSubConnStateChange: 0xc0005ddc00, READY
 DEBUG Finished unary call                      duration=44.494674ms grpc_method=Create grpc_service=ttn.lorawan.v3.EndDeviceRegistry namespace=grpc
 DEBUG Set end device on Network Server         paths=[frequency_plan_id lorawan_phy_version lorawan_version multicast session.dev_addr session.keys.f_nwk_s_int_key.key supports_join]
 DEBUG Finished unary call                      duration=455.538557ms grpc_method=Set grpc_service=ttn.lorawan.v3.NsEndDeviceRegistry namespace=grpc
 DEBUG Set end device on Application Server     paths=[session.dev_addr session.keys.app_s_key.key]
 DEBUG Finished unary call                      duration=48.61551ms grpc_method=Set grpc_service=ttn.lorawan.v3.AsEndDeviceRegistry namespace=grpc
{
  "ids": {
    "device_id": "dev2",
    "application_ids": {
      "application_id": "app1"
    }
  },
  "created_at": "2019-07-01T08:11:48.868Z",
  "updated_at": "2019-07-01T08:11:49.381217500Z",
  "attributes": {
  },
  "network_server_address": "localhost",
  "application_server_address": "localhost",
  "lorawan_version": "1.0.2",
  "lorawan_phy_version": "1.0.2-b",
  "frequency_plan_id": "EU_863_870",
  "session": {
    "dev_addr": "00E4304D",
    "keys": {
      "f_nwk_s_int_key": {
        "key": "B7F3E161BC9D4388E6C788A0C547F255"
      },
      "app_s_key": {
        "key": "A0CAD5A30036DBE03096EB67CA975BAA"
      }
    },
    "started_at": "0001-01-01T00:00:00Z"
  },
  "multicast": true
}
```